src/utils/string.js

```diff
@@ -197,7 +198,14 @@ function toHex(buffer) {
  */
 export function hash(stringToHash) {
   const crypto = self.crypto || self.msCrypto;
-  const subtle = crypto.subtle;
+  const subtle = crypto?.subtle;
```

Comment:
it would've been nice if self.crypto/self.msCrypto were typed properly so the previous code triggered a JS compiler error instead.

Reply:
Yep, that's definitely a gap in the type system